assets/js/wcmp-frontend.js

```diff
@@ -1,138 +1,158 @@
 jQuery(function($) {
-    window.myparcel_checkout_updating = false;
-    window.myparcel_force_update = false;
-
-    window.myparcel_selected_shipping_method = '';
-    window.myparcel_updated_shipping_method = '';
-    window.myparcel_selected_country = '';
-    window.myparcel_updated_country = '';
-
-    window.myparcel_is_using_split_address_fields = wcmp_display_settings.isUsingSplitAddressFields;
-    window.myparcel_shipping_methods = JSON.parse(wcmp_delivery_options.shipping_methods);
-    window.myparcel_always_display = wcmp_delivery_options.always_display;
-
-    // hide checkout options for non parcel shipments
-    $(document).on('updated_checkout', function() {
-        window.myparcel_checkout_updating = false; //done updating
-
-        if (window.myparcel_always_display) {
-            window.myparcel_force_update = true;
-            show_myparcel_delivery_options();
-        } else if (window.myparcel_shipping_methods.length > 0) {
-            var shipping_method;
-            window.myparcel_selected_country = window.myparcel_updated_country;
+    var MyParcel_Frontend = {
```

Comment:
MyParcelFrontend, myparcel_frontend of myParcelFrontend

Reply:
Global "MyParcel" heet ook zo, vandaar dat ik hiervoor heb gekozen